tests/integrations/storage/test_storage.rs

```diff
@@ -698,7 +704,14 @@ pub fn test_txn_store_gc_multiple_keys_single_storage(n: usize, prefix: String)
         store.put_ok(k.as_bytes(), b"v1", 5, 10);
         store.put_ok(k.as_bytes(), b"v2", 15, 20);
     }
-    store.gc_ok(30);
+
+    let store_id = 1;
+    let mut region = metapb::Region::default();
+    region.set_peers(RepeatedField::from_vec(vec![metapb::Peer {
```

Comment:
You mean `region.mut_peers().push(new_peer(store_id, 0))`?

There are still many places using `RepeatedField`.

Reply:
Done